proto/desmos/subspaces/v1beta1/genesis.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "desmos/subspaces/v1beta1/subspace.proto";
+import "desmos/subspaces/v1beta1/models.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// GenesisState contains the data of the genesis state for the subspaces module
+message GenesisState {
+  repeated desmos.subspaces.v1beta1.Subspace subspaces = 1
+        [ (gogoproto.nullable) = false ];
+  repeated SubspaceAdminsEntry subspace_admins = 2
+        [ (gogoproto.nullable) = false ];
+  repeated BlockedUsersEntry blocked_to_post_users = 3
+        [ (gogoproto.nullable) = false ];
+}
```

Comment:
Since the admins and allowed users won't change that much, can we have those data directly inside a subspace? I think it would be much easier to handle as well as better for anyone querying a subspace to get all the data directly. 

This would make the genesis state a lot simpler: 

```protobuf
message GenesisState {
  repeated Subspaces subspaces = 1
}
```